Ports/acpica-tools/package.sh

```diff
@@ -6,12 +6,11 @@ files=(
     "https://github.com/acpica/acpica/archive/refs/tags/${version}.tar.gz 2248799b7ca08a7711ac87d31924354ed49047507607d033bd327ba861ec4d31"
 )
 
-
-build() {
-    run make iasl
-    run make acpixtract
-    # FIXME: Make "run make acpiexec" to work
-    run make acpihelp
-    run make acpisrc
-    run make acpibin
-}
+_targets=(
+    iasl
+    acpixtract
+    acpihelp
+    acpisrc
+    acpibin
+)
+makeopts+=("${_targets[@]}")
```

Comment:
Why are we doing this? This doesn't look like it's related to warning flags. `_targets` isn't used anywhere else, so we could just add them to `makeopts` directly.

Also, the FIXME is now missing.

Reply:
It was commented out, do you want to me add that target?

Reply:
No, I assume there was a reason why it wasn't added when the port was updated. But I still want you to keep the FIXME comment.

And still, I don't see the reason why we are putting all the targets into an array here, considering that this array doesn't seem to be used elsewhere (other than appending all the targets to `makeopts`).

Reply:
Yes it makes sense I will move it to `makeopts`.